src/mbgl/map/source.hpp

```diff
@@ -46,6 +47,7 @@ class SourceInfo : private util::noncopyable {
     std::string attribution;
     std::array<float, 3> center = {{0, 0, 0}};
     std::array<float, 4> bounds = {{-180, -90, 180, 90}};
+    std::string source_id = "";
```

Comment:
Was keeping in line with the other properties here like `min_zoom` and `tile_size`.